elastic/logs/challenges/esql.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "esql",
```

Comment:
This challenge is currently only used for easier testing on an already populated dataset. The plan is to delete this before merging.

If, however, we want to keep ESQL in a separate challenge, a lot more work is required:

* Copy over the index setup and bulk indexing code from `logging-querying`
*. Copy the nightlies config for `logging-querying` to a new `elastic-logs-esql` config
* Simplify the above two setup for anything not really required for `esql`

And finally, such a step would mean duplicating the benchmark activity of generating the data, which is currently a very time-consuming and expensive step. It takes many hours, and the `eqsl` queries take only minutes to run. It is much more resource efficient to just run after `logging-querying` on the same challenge.

Reply:
I like adding `esql` at the end of `logging-querying`.